src/main/fc/fc_msp.c

```diff
@@ -1140,7 +1140,7 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         sbufWriteU8(dst, motorConfig()->useUnsyncedPwm);
         sbufWriteU8(dst, motorConfig()->motorPwmProtocol);
         sbufWriteU16(dst, motorConfig()->motorPwmRate);
-        sbufWriteU16(dst, (uint16_t)(motorConfig()->digitalIdleOffsetPercent * 100));
+        sbufWriteU16(dst, (uint16_t)round(motorConfig()->digitalIdleOffsetPercent * 100));
```

Comment:
Should probably change this to `lrintf` as well, to be in line with the rest. I just noticed that truncating was giving off by one results in some cases.